src/SFA.DAS.AANHub.Data/AANDataContext.cs

```diff
@@ -1,29 +1,30 @@
-﻿using System.Diagnostics.CodeAnalysis;
-using Microsoft.EntityFrameworkCore;
+﻿using Microsoft.EntityFrameworkCore;
 using SFA.DAS.AANHub.Domain.Entities;
 using SFA.DAS.AANHub.Domain.Interfaces;
+using SFA.DAS.AANHub.Domain.Models;
+using System.Diagnostics.CodeAnalysis;
 
-namespace SFA.DAS.AANHub.Data
+namespace SFA.DAS.AANHub.Data;
+
+[ExcludeFromCodeCoverage]
+public class AanDataContext : DbContext, IAanDataContext
 {
-    [ExcludeFromCodeCoverage]
-    public class AanDataContext : DbContext, IAanDataContext
-    {
-        public DbSet<Audit> Audits => Set<Audit>();
-        public DbSet<Region> Regions => Set<Region>();
-        public DbSet<Member> Members => Set<Member>();
-        public DbSet<Apprentice> Apprentices => Set<Apprentice>();
-        public DbSet<Employer> Employers => Set<Employer>();
-        public DbSet<Partner> Partners => Set<Partner>();
-        public DbSet<Admin> Admins => Set<Admin>();
-        public DbSet<StagedApprentice> StagedApprentices => Set<StagedApprentice>();
-        public DbSet<Calendar> Calendars => Set<Calendar>();
-        public DbSet<Profile> Profiles => Set<Profile>();
-        public AanDataContext(DbContextOptions<AanDataContext> options) : base(options) { }
+    public DbSet<Audit> Audits => Set<Audit>();
+    public DbSet<Region> Regions => Set<Region>();
+    public DbSet<Member> Members => Set<Member>();
+    public DbSet<Apprentice> Apprentices => Set<Apprentice>();
+    public DbSet<Employer> Employers => Set<Employer>();
+    public DbSet<Partner> Partners => Set<Partner>();
+    public DbSet<Admin> Admins => Set<Admin>();
+    public DbSet<StagedApprentice> StagedApprentices => Set<StagedApprentice>();
+    public DbSet<Calendar> Calendars => Set<Calendar>();
+    public DbSet<Profile> Profiles => Set<Profile>();
+    public DbSet<CalendarEventModel>? CalendarEventModel { get; set; }
```

Comment:
CalendarEvents

Reply:
Can;'t call it that because there is an entity 'CalendarEvent' with CalendarEvents associated, this is a mapped model.  Will call it CalendarEventModels.